src/python/pants/backend/project_info/tasks/dependencies.py

```diff
@@ -38,6 +45,19 @@ def __init__(self, *args, **kwargs):
       raise TaskError('At most one of --internal-only or --external-only can be selected.')
 
   def console_output(self, unused_method_argument):
+    deprecated_conditional(
+      lambda: not self.is_external_only and not self.is_internal_only,
+      removal_version="1.26.0.dev1",
+      entity_description="External dependencies will no longer be included in dependencies output.",
+      hint_message="Pants will soon default to `--internal-only`, and remove the `--external-only` option. "
+                    "and return only target addresses. Currently, Pants defaults to include both internal "
```

Comment:
Don't need the comma. Also note you end the sentence, but the next line continues with `and return only target addresses.` You can remove that second part.